Ivanti Pulse Secure Mapping (#27407)
demisto/content: Packs/IvantiPulseSecureVTM/README.md

# Ivanti Pulse Secure VTM
This pack includes Cortex XSIAM content. 
<~XSIAM>
## Configuration on Server Side
You need to configure the Virtual Traffic Manager (VTM) to forward Syslog messages.

Open the VTM UI; 

For forwarding Event logs:
1. Open the **System** tab, then, click on **Alerting**.
2. At the **Alert Mappings** section, make sure Audit Events are kept in Syslog (action).
3. Next, open the **Global Settings** tab, then, click on **Logging**.
4. Mark the "Whether to mirror the audit log to EventID" checkbox as **Yes**.
5. Under the **Apply Changes** section, press on **Update** once you are done.

Setup for forwarding requests logging per Virtual Server (VS):

> 1. Open the **Services** tab, Under **Virtual Servers**, select a VS.

Reviewer: ```suggestion
1. Click the **Services** tab, and under **Virtual Servers**, select a VS.
```

2. Press on the **Request Logging** section for the relevant VS.
3. Under **Remote Request Logging**, make sure to preform the following;
    3.1. Mark the checkbox for **syslog!enabled** as **Yes**.
    3.2. At the **syslog!ipendpoint** section, fill the remote IP and Port of your syslog collector.

>     3.3. At the **syslog!msg_len_ limit** section, set the limit for a syslog message to **2048**.

Reviewer: ```suggestion
    3.3. In the **syslog!msg_len_ limit** section, set the limit for a syslog message to **2048**.
```

>     3.4. At the **syslog!format** section, set the relevant syslog format

Reviewer: ```suggestion
    3.4. In the **syslog!format** section, set the relevant syslog format.
```

       * For HTTPS traffic inspection based VSs, select the **Simple connection log** format.
       * For HTTP traffic inspection based VSs, select the **custom** format, input-
         ```bash
         %t|%T|%h|%m %U|%{Content-Type}o|%s|%u|%b|%{Cookie}i|%{Referer}i|%{User-Agent}i
         ``` 

> 4. Under the **Apply Changes** section, press on **Update** once you are done. 

Reviewer: ```suggestion
4. In the **Apply Changes** section, click **Update**. 
```
